### PR TITLE
Add sensors support for wt86518 and wt86528

### PR DESCRIPTION
**Pull request by jenneron**

### CODE REVIEW

**Comment by Minecrell** [comment]
Downstream has the same sensors listed for both a6000 and a6010. Is it possible that they used different sensors for both devices? (Would need to ask some other people with a6000/a6010 to figure that out).

**Comment by jenneron** [comment]
> Downstream has the same sensors listed for both a6000 and a6010. Is it possible that they used different sensors for both devices? (Would need to ask some other people with a6000/a6010 to figure that out).

Downstream dmesg from a6000:
```
<3>[    1.859505] i2c-msm-v2 78b6000.i2c: NACK: slave not responding, ensure its powered, Invalid slave addr, I2C transfer failed, : msgs(n:2 cur:0 tx) bc(rx:1 tx:1) mode:FIFO slv_addr:0x68 MSTR_STS:0x0d1343c8 O>
<3>[    1.859534] mpu6050 0-0068: Fail to read power mode, ret=-107
<3>[    1.859541] mpu6050 0-0068: Cannot get invalid chip type
<3>[    1.859587] mpu6050 0-0068: Probe device return error-107
<4>[    1.859602] mpu6050: probe of 0-0068 failed with error -107
...
<3>[    1.859852] akm09911 0-000c: akm_i2c_txdata: transfer failed.
<3>[    1.859858] akm09911 0-000c: AKECS_Set_PowerDown: Can not set to powerdown mode.
<3>[    1.949494] i2c-msm-v2 78b6000.i2c: NACK: slave not responding, ensure its powered, Invalid slave addr, I2C transfer failed, : msgs(n:2 cur:0 tx) bc(rx:2 tx:1) mode:FIFO slv_addr:0xc MSTR_STS:0x0d1343c8 OP>
<3>[    1.949520] akm09911 0-000c: akm_i2c_rxdata: transfer failed.
<4>[    1.949572] akm09911: probe of 0-000c failed with error -107

```
So mpu6050 and ak09911 are not used in a6000. It has kxcj9 accelerometer, but doesn't seem to have a magnetometer.

a6000 downstream dmesg - https://dpaste.com/8Q8BSTTXY
a6010 downstream dmesg - https://dpaste.com/DWJTQMFNE
(a6010 log is a little cut off, but includes sensors probing)

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts [hunk 1]
```diff
@@ -31,3 +51,13 @@
 &usb_hs_phy {
 	extcon = <&usb_vbus>;
 };
+
+&msmgpio {
+	accelerometer_default: accelerometer-default {
+		pins = "gpio112", "gpio114";
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts:57]: This looks like weird copy paste

**Review by Minecrell — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts (hunk 1 above)
Comment (jenneron) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts:57]: What should be there? I didn't find kxcj9 in the schematics.

Comment (Minecrell) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts:57]: GPIO81 normally, but according to the schematics that goes to something entirely different. So perhaps just remove the interrupt.

**Review by Minecrell — APPROVED** [review]

**Pull request merged**